primefaces-showcase/src/main/java/org/primefaces/showcase/view/data/dataexporter/CustomizedDocumentsView.java

```diff
@@ -122,10 +120,10 @@ public void setService(ProductService service) {
         this.service = service;
     }
 
-    public void postProcessXLS(Object document) {
+    public void onRowExportXLS(Object document) {
```

Comment:
the method originally was to hightlight the header
now you hightlight the all rows, right?

shouldnt we create a different method/example here?

Reply:
Yeah i'm gonna change that, so it looks like strippedRows showcase